docs/tempo/website/traceql/construct-query.md

```diff
@@ -0,0 +1,276 @@
+
+Query results are returned in a table. Selecting the Trace ID or Span ID provides more detailed information.
+
+<p align="center"><img src="../assets/query-editor-results-span.png" alt="Query editor showing span results" /></p>
+
+Selecting the trace ID from the returned results will open a trace diagram. Selecting a span from the returned results opens a trace diagram and reveals the relevant span in the trace diagram (above, the highlighted blue line).
+
+## Selecting spans
+
+In TraceQL, curly brackets `{}` always select a set of spans from the current trace. They are commonly paired with a condition to reduce the spans being passed in.
+
+This simple query will be evaluated on every span of every trace, one at a time.
+
+```
+{ .http.status = 200 }
+```
+
+If the trace being evaluated contains no spans with an attribute `http.status` with the value `200`, then no spans will be selected and this trace will not appear in the result set.
+
+If the trace does contain spans with an attribute `http.status` with the value `200`, then only those spans will be returned. The trace is reduced to only the set of spans that match the condition inside the `{}`. The result set will contain only this subset of spans matching the condition.
+
+### Intrinsic fields
+
+Intrinsic fields are fundamental to spans. Thes fields can be referenced when selecting spans.
+
+
+| **Operation** | **Type** | **Definition**                        | **Example**            |
+|---------------|----------|---------------------------------------|------------------------|
+| status        | string   | status values are error, ok, or unset | { status = ok }        |
+| duration      | duration | end - start time of the span          | { duration > 100ms }   |
+| name          | string   | operation or span name                | { name = "HTTP POST" } |
+
+### Attribute fields
+
+Attribute fields are derived from the span and can be customized. Process and span attribute types are [defined by the attribute itself](https://github.com/open-telemetry/opentelemetry-proto/blob/b43e9b18b76abf3ee040164b55b9c355217151f3/opentelemetry/proto/common/v1/common.proto#L30-L38), whereas intrinsic fields have a built-in type. You can refer to dynamic attributes (also known as tags) on the span or the span's resource.
+
+Attributes in a query start with a period and must end with a space. To find traces with the `GET HTTP` method, your query could look like this:
+
+```
+{ .http.method = "GET" }
+```
+
+Or like this:
+
+```
+{.http.method ="GET"}
+```
+
+#### Examples
+
+Find traces that passed through the prod namespace:
+
+```
+{ .namespace = "prod" }
+```
+
+Find any database connection string that goes to a Postgres or MySQL database:
+
+```
+{ .db.system =~ "postgresql|mysql" }
+```
+
+### Scoped attribute fields
+
+Attributes can be specifically scoped to either "span" or "resource". Specifying "span" or "resource" can result in significant performance benefits.
```

Comment:
I would almost prefer listing this first to encourage people to think of traceql this way. Then later noting that `.` is a wildcard that searches both `span.` and `resource.`